.github/workflows/propagate_to_manifestdb.yml

```diff
@@ -0,0 +1,77 @@
+name: Propagate update to OSBuild
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  propagate:
+    runs-on: ubuntu-latest
+    env:
+      SCHUTZBOT_GH: ${{ secrets.SCHUTZBOT_GH}}
+    steps:
+      - name: Checkout OSBuild
+        uses: actions/checkout@v3
+        with:
+          path: osbuild
+          fetch-depth: 0
+
+      - name: Update Manifest-db
+        run: |
+          # update only if manifest-db SHA was updated in the last commit
+          cd osbuild
+          ACTUAL_MDB_SHA=$(git show HEAD:Schutzfile | jq -r '.global.dependencies."manifest-db".commit')
+          PREVIOUS_MDB_SHA=$(git show HEAD^:Schutzfile | jq -r '.global.dependencies."manifest-db".commit')
+          if [[ $ACTUAL_MDB_SHA != $PREVIOUS_MDB_SHA ]] ; then
+
+            echo "updating the dependency in manifest-db"
+            OSBUILD_HEAD="${{ github.sha }}"
+            echo "Latest OSBuild SHA: $OSBUILD_HEAD"
+
+            # login as schutzbot
+            echo "${SCHUTZBOT_GH}" | gh auth login --with-token
```

Comment:
I think this should be moved right before `gh pr create`. It feels very out of context here.